contact-us/general-inquiries.html

```diff
@@ -7,37 +7,54 @@ <h1 class="h4">
 <p class="short-desc">
     {{ general_inqueries.sitewide_desc }}
 </p>
-<ul class="list list__icons">
-{%- if general_inqueries.email_0_addr %}
-    <li class="list_item u-break-word">
-        <span class="cf-icon cf-icon-email list_icon"></span>
-        <span class="h5 list_text">Email:</span>
-        <p>
-            <a class="list_link" href="mailto:{{ general_inqueries.email_0_addr }}">
-                {{ general_inqueries.email_0_addr }}
-            </a>
-        </p>
-    </li>
-{%- endif %}
+{%- if general_inqueries.email or if general_inqueries.phone or if general_inqueries.fax_num %}
+    <ul class="list list__icons">
+    {%- if general_inqueries.email %}
+        {%- set first = True %}
+        {%- for email in general_inqueries.email %}
+            {%- if email.addr %}
+                <li class="list_item u-break-word">
+                    {%- if first %}
+                        <span class="cf-icon cf-icon-email list_icon"></span>
+                        {%- set first = False %}
+                    {%- endif %}
+                    <span class="h5 list_text">Email:</span>
+                    <p>
+                        <a class="list_link" href="mailto:{{ email.addr }}">
+                            {{ email.addr }}
+                        </a>
+                    </p>
+                </li>
+            {%- endif %}
+        {%- endfor %}
+    {%- endif %}
 
-{%- if general_inqueries.phone_0_num %}
-    <li class="list_item">
-        <span class="cf-icon cf-icon-phone list_icon"></span>
-        <span class="h5 list_text">Phone:</span>
-        <p>
-            <a class="list_link list_link__phone" href="tel:{{ general_inqueries.phone_0_num }}">
-            {{ format_phone(general_inqueries.phone_0_num) }}
-            </a>
-        </p>
-    </li>
-{%- endif %}
-{%- if general_inqueries.fax_num %}
-    <li class="list_item">
-        <span class="cf-icon cf-icon-fax list_icon"></span>
-        <span class="h5 list_text">Fax:</span>
-        <p class="short-desc">
-            {{ format_phone(general_inqueries.fax_num) }}
-        </p>
-    </li>
-{%- endif %}
+    {%- if general_inqueries.phone %}
+        {%- set first = True %}
+        {%- for phone in general_inqueries.phone %}
+            {%- if phone.num %}
+                <li class="list_item">
+                    {%- if first %}
```

Comment:
Another (possibly more elegant) solution would be to check the `{{ loop.index }}` (will be 0 for the first one).  This applies to a number of these first = True situations.